doc/modules/linear_model.rst

```diff
@@ -920,6 +923,13 @@ controls the strength of :math:`\ell_1` regularization vs. :math:`\ell_2`
 regularization. Elastic-Net is equivalent to :math:`\ell_1` when
 :math:`\rho = 1` and equivalent to :math:`\ell_2` when :math:`\rho=0`.
 
+Note that the scale of the class weights and the sample weights will influence
+the optimization problem. For instance, multiplying the sample weights by a
+constant :math:`b>0` is equivalent to multiplying the (inverse) regularization
+strength `C` by :math:`b`. This is because user defined sample and class
+weights are multiplied to the coefficients and regularization penalizes large
+coefficients by adding a cost to the loss.
```

Comment:
```suggestion
strength `C` by :math:`b`.
```
I feel bad to also ask to also remove this sentence. It's just not 100% correct. From the optimization problem one sees that `C` could be drawn insight the sum and "integrated" into the weights $s_i$. Weights are never multiplied to coefficients, only to the per sample losses.

Reply:
Don't feel bad, @lorentzenchr. This it why I had wanted to do this issue.